src/stores/video.ts

```diff
@@ -57,13 +57,95 @@ export const useVideoStore = defineStore('video', () => {
   const timeNow = useTimestamp({ interval: 500 })
   const autoProcessVideos = useBlueOsStorage('cockpit-auto-process-videos', true)
   const lastRenamedStreamName = ref('')
+  const streamsPreparingToRecord = ref<string[]>([])
 
   const namesAvailableStreams = computed(() => mainWebRTCManager.availableStreams.value.map((stream) => stream.name))
 
+  const getRecorderOptions = async (streamData: StreamData): Promise<MediaRecorderOptions> => {
+    // Check on WebRTCManager to exposed RTCPeerConnection details.
+    const waitForPeerConnection = async (): Promise<RTCPeerConnection | null> => {
```

Comment:
Since this function is very simple and only used here, can we make it not a function, and just code it inside the context?